requirements/edx/base.txt

```diff
@@ -551,6 +560,8 @@ fastavro==1.9.5
     # via openedx-events
 filelock==3.15.4
     # via snowflake-connector-python
+firebase-admin==6.5.0
+    # via edx-ace
```

Comment:
Not your fault @dianakhuang , but I am now realizing that the new version of edx-ace is pulling in an absolute pile of new google dependencies 😓 

I am going to run this by my team and see whether or not those new deps are an expected outcome of the recent mobile notifications project. I should know by 2pm whether we're going to let these transitive deps in, or whether we're going to pin+fix edx-ace.

Up to you whether you want to make this PR just pin social-auth-app-django vs. wait for me while I figure out this edx-ace thing.

Reply:
SGTM @dianakhuang and thanks for the quick turnaround on the pin. In that case my only other request is that you make this a `build:` rather than a `feat:` .